nomic/aws/__init__.py

```diff
@@ -0,0 +1,11 @@
+def _check_aws_dependencies():
+    try:
+        import tokenizers
+        import tritonclient
+    except ImportError as e:
+        missing_package = str(e).split("No module named ")[-1].strip("'")
+        raise ImportError(f"The '{missing_package}' package is required for this feature. "
+                          "Please install it by running 'pip install nomic[aws]'.")
+    
+_check_aws_dependencies()
```

Comment:
is this always run?

Reply:
only on importing this submodule

Reply:
import nomic will work fine
import nomic.aws will raise the ImportError